### PR TITLE
feat: Change state and show live data

### LINKED ISSUE #7: feat: Connect top page to server

### LINKED ISSUE #31: feat: Change state and show live data

### PR DESCRIPTION
**Pull request by marimarimarietty**

### CODE REVIEW

File: src/components/organisms/Form.tsx [hunk 1]
```diff
@@ -169,7 +173,7 @@ const CheckLink = styled("a", {
   "&:visited": { color: indigo.indigo11 },
 });
 
-const Form = () => (
+const Form = ({ setLoggedin }: Props) => (
   <Box css={{}}>
     <Tabs defaultValue="tab1">
```
Comment (Takashicc) [inline src/components/organisms/Form.tsx]: Is this css required? `css={{}}`

File: src/components/organisms/Top.tsx [hunk 2]
```diff
@@ -31,6 +34,13 @@ export default function Top() {
       .then((res) => {
         setWeatherData(res.data);
       });
+
+    // live data
+    axios
+    .get(`${process.env.NEXT_PUBLIC_SERVER_URL}/device-data/${deviceId}/live`)
+    .then((res) => {
+      setLiveData(res.data);
+    });
```
Comment (Takashicc) [inline src/components/organisms/Top.tsx:44]: Add `// TODO Implement catch`

**Review by Takashicc — CHANGES_REQUESTED** [review]

File: src/components/organisms/Form.tsx (hunk 1 above)
Comment (marimarimarietty) [reply src/components/organisms/Form.tsx]: We don't need that.

**Review by Takashicc — APPROVED** [review]
LGTM!

**Pull request merged**